src/main/java/de/adesso/anki/battle/providers/VehicleStateProvider.java

```diff
@@ -72,6 +73,21 @@
 			//generating  facts for vehicle 
 			//body all other in the current world
 			String obstacleType = body.getClass().getSimpleName() ;
+			
+			double offSetFromCenter;
+			if (obstacleType.equals("Rocket")) {
+				//rocket has no track attribute
+				offSetFromCenter = 0 ; 

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this useless assignment to local variable "offSetFromCenter". ([squid:S1854](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1854))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=24&issues=AWLTnu90jr7wspRorFgm&open=AWLTnu90jr7wspRorFgm)

[ISSUE_KEY]: AWLTnu90jr7wspRorFgm